<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivacySettings</class>
 <widget class="QWidget" name="PrivacySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>280</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="cbTypingNotification">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Typing Notification</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbKeepHistory">
         <property name="toolTip">
          <string extracomment="History keeping still under developing. Log format changin is possible."/>
         </property>
         <property name="text">
          <string>Keep History (unstable)</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="encryptionGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Encryption</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="cbEncryptTox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Encrypt Tox datafile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbEncryptHistory">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Encrypt History</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="nospamGroup">
         <property name="title">
          <string>Nospam</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="nospamLineEdit">
              <property name="inputMask">
               <string>HHHHHHHH</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="randomNosapamButton">
              <property name="text">
               <string>Generate random nospam</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
